### PR TITLE
Catalog + Ansible sources UX enhancements

### PR DESCRIPTION
**Pull request by rvsia**
TPINVTRY-880

Configure credentials

image

HTTPs placeholder

image

Verify SSL true

image

Verify SSL false

image

### CODE REVIEW

File: packages/sources/src/addSourceWizard/schemaBuilder.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ export const createEndpointFlagger = (skipEndpoint) => ({
     component: componentTypes.TEXT_FIELD,
     name: 'noEndpoint',
     hideField: true,
-    initialValue: skipEndpoint || '',
+    initialValue: skipEndpoint ? true : false,
```
Comment (Hyperkid123) [inline packages/sources/src/addSourceWizard/schemaBuilder.js]: `Bool(skipEndpoint)` nebo `!!skipEndpoint`

File: packages/sources/src/sourceFormRenderer/components/SourceWizardSummary.js [hunk 2]
```diff
@@ -56,18 +59,22 @@ const SourceWizardSummary = ({ sourceTypes, formOptions, applicationTypes, showA
...
 
     const fields = [ ...authTypeFields, ...endpointFields ];
 
     const application = values.application ? applicationTypes.find(type => type.id === values.application.application_type_id) : undefined;
-    const applicationName = application ? application.display_name : 'Not selected';
 
-    const valuesInfo = [ ...flattenDeep(allValuesPath(values, fields)) ].filter(x => x);
+    const { display_name = 'Not selected', name, id } = application ? application : {};
+
+    let availableStepKeys = getStepKeys(type.name, hasAuthentication, name, id);
```
Comment (Hyperkid123) [inline packages/sources/src/sourceFormRenderer/components/SourceWizardSummary.js]: `const`

**Comment by rvsia** [comment]
@Hyperkid123 Tested with the latest DDF and it works.

**Comment by Hyperkid123** [comment]
@rvsia snapshots

**Pull request merged**